invenio_rdm_records/services/access/service.py

```diff
@@ -407,6 +414,28 @@ def read_grant(self, identity, id_, grant_id, expand=False):
             expand=expand,
         )
 
+    def read_grant_by_user_id(self, identity, id_, user_id, expand=False):
+        """Read a specific access grant of a record by user id."""
+        record, parent = self.get_parent_and_record_or_draft(id_)
+
+        # Permissions
+        self.require_permission(identity, "manage", record=record)
```

Comment:
I'm not sure, what would be the difference?

Reply:
It might be that @0einstein0 refers to splitting who can read and who can manage, if needed in the future.
However, I would say that is quite easy to change later on, given that all this is internal in the backend.

Reply:
@ntarocco in the UI - no, the button that opens the modal to see and manage the grants appears only to those who `can_manage`. In the rest api I would say that this information also shouldn't be available to everyone